lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
```

Comment:
Should we add some user context as well? Like h does: https://github.com/hypothesis/h/blob/master/h/sentry.py#L35-L39

Not sure if you have any user identifying data on the request that might be interesting to capture.